projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java

```diff
@@ -97,7 +97,12 @@ public String startDataJobExecution(
               ? jobExecutionRequest.getStartedBy() + "/" + operationContext.getUser()
               : operationContext.getUser();
       annotations.put(JobAnnotation.STARTED_BY.getValue(), startedBy);
-      annotations.put(JobAnnotation.EXECUTION_TYPE.getValue(), ExecutionType.MANUAL.getValue());
+      annotations.put(
+          JobAnnotation.EXECUTION_TYPE.getValue(),
```

Comment:
Considering how much we discuss it. Add a comment explaining why do we do it this way for future mantainers.

Reply:
Added the following comment:
```
// 'Manual' executions must have their `startedBy` follow the structure 'manual/*triggering-mechanism*'
// For example, executions started from the CLI would have `startedBy` equal to 'manual/vdk-control-cli'
// Additionally we also check if it contains 'vdk-control-cli' here for the purposes of backwards compatibility
```